bots/running-a-rasa-bot/README.md

```diff
@@ -1,54 +1,33 @@
 # Running a Rasa Bot
```

Comment:
Please check my PR here and refactor all the hedings according to the structure:
https://github.com/RocketChat/docs/pull/1445/files